plugins/services/src/js/data/groups/__tests__/index-test.ts

```diff
@@ -121,6 +121,144 @@ describe("Services Data Layer - Groups", () => {
         m.expect(dl.query(query, {}).pipe(take(1))).toBeObservable(expected$);
       })
     );
+
+    it(
+      "handles query for filtering enforce quota",
+      marbles(m => {
+        const marathonServiceTree = makeServiceTree(marathonGroups);
+        const roles$ = m.cold("(a|)", {
```

Comment:
no action items here, just curious:
why do we mock this with a marble? wouldn't `of` be more straightforward?